src/audio/host.c

```diff
@@ -715,13 +742,14 @@ static int host_copy(struct comp_dev *dev)
 		if (ret < 0) {
 			if (ret == -ENODATA) {
 				/* preload not finished, so stop processing */
-				trace_host("host_copy(), preload not yet "
-					   "finished");
+				trace_host_with_ids(dev, "host_copy(), preload"
+						    " not yet finished");
 				return PPL_STATUS_PATH_STOP;
 			}
 
-			trace_host_error("host_copy() error: dma_copy() "
-					 "failed, ret = %u", ret);
+			trace_host_error_with_ids(dev, "host_copy() error: "
+						  "dma_copy() failed, ret = %u"
+						  , ret);
```

Comment:
Comma in previous line.

Reply:
I've moved "ret = %u" to the next line, since ""dma_copy() failed, ret = %u," would exceed checkpatch line character limit.